src/Select/Select.js

```diff
@@ -0,0 +1,188 @@
+
+export const SIZE_OPTIONS = {
+  SMALL: "small",
+  NORMAL: "normal",
+};
+
+type Option = {
+  value: string | number,
+  label?: string,
+  disabled?: boolean,
+  visible?: boolean, // eslint-disable-line react/no-unused-prop-types
+};
+
+type Props = {
+  size: $Values<typeof SIZE_OPTIONS>,
+  label?: string,
+  placeholder?: string,
+  value: string,
+  disabled?: boolean,
+  error?: React.Node,
+  help?: React.Node,
+  onChange: (SyntheticInputEvent<HTMLSelectElement>) => any,
+  options: Option[],
+  prefix?: React.Node,
+  theme: typeof defaultTokens,
+};
+
+type State = {
+  value: string,
+};
+
+const SelectContainer = styled(({ theme, ...props }) => <div {...props} />)`
+  position: relative;
+  display: flex;
+  align-items: center;
+  border-radius: ${({ theme }) => theme.borderRadiusNormal};
+  background: ${({ theme }) => theme.backgroundInput};
+  border: solid 1px
+    ${({ theme, error }) => (error ? theme.borderColorInputError : theme.borderColorInput)};
+  width: 100%;
+  box-sizing: border-box;
+  cursor: pointer;
+  &:hover {
+    border-color: ${({ theme, error }) =>
+      error ? theme.paletteRedNormalActive : theme.borderColorInputHover};
+  }
+  &:focus-within {
+    border-color: ${({ theme, error }) =>
+      error ? theme.borderColorInputError : theme.borderColorInputFocus};
+    box-shadow: ${({ theme, error }) =>
+      error ? theme.boxShadowInputErrorFocus : theme.boxShadowInputFocus};
+  }
+`;
+
+const SelectPrefix = styled(({ theme, prefix, ...props }) => <div {...props} />)`
+  display: flex;
+  align-items: center;
+  position: absolute;
+  padding: ${({ theme }) => `0 ${theme.spaceSmall}`};
+  pointer-events: none;
+`;
+
+const SelectSizeCondition = (theme, size) => {
+  if (size === SIZE_OPTIONS.SMALL) {
+    return `${theme.spaceXSmall} ${theme.spaceXLarge} ${theme.spaceXSmall} ${theme.spaceSmall}`;
+  }
+  return `${theme.spaceSmall} ${theme.spaceXXLarge} ${theme.spaceSmall} ${theme.spaceMedium}`;
+};
+
+const StyledSelect = styled(({ theme, prefix, ...props }) => <select {...props} required />)`
+  width: 100%;
+  height: 100%;
+  border: none;
+  border-radius: ${({ theme }) => theme.borderRadiusNormal};
+  font-family: ${({ theme }) => theme.fontFamily};
+  font-size: ${({ theme, size }) =>
+    size === SIZE_OPTIONS.SMALL ? theme.fontSizeInputSmall : theme.fontSizeInputNormal};
+  color: ${({ theme }) => theme.colorTextInput};
+  background: ${({ theme }) => theme.backgroundInput};
+  padding: ${({ theme, size }) => SelectSizeCondition(theme, size)};
+  padding-left: ${({ prefix }) => prefix && "48px"}; // temporary replacement for token
+  appearance: none;
+  outline: none;
+  cursor: pointer;
+  /* for IE10 */
+  &::-ms-expand {
+    display: none;
+  }
+  &:disabled {
+    color: ${({ theme }) => theme.colorTextInputDisabled};
+    background: ${({ theme }) => theme.backgroundInputDisabled};
+    cursor: default;
+  }
+  /* placeholder */
+  &:invalid {
```

Comment:
Please for `placeholder` style use another way. Required attr will probably render native browser error. Do this like:
```js
const isPlaceholder = this.props.placeholder && this.state.value === ""
```
You then pass this const to `StyledSelect` and it'll decide which color should apply.